src/composable/types.ts

```diff
@@ -98,9 +98,9 @@ export type PollParams = OwnerContext & {
   blockInfo?: BlockInfo
 
   /**
-   * Allows to optional pass the config of the orderbook API
+   * Allows to optionally pass the config for the OrderBook API, or a pre-configured API object
    */
-  orderbookApiConfig?: OrderBookApiConfig
+  orderBookOrConfig?: OrderBookApi | OrderBookApiConfig
```

Comment:
I'm not 100% certain whether this is a major version bump needed kind of change.
I say this because, although the type is changing, the old type content is still supported.
So previous usage should still work as long as the type is not used directly.

Reply:
The type is used by downstream (`watch-tower`) :eyes: